### ISSUE TITLE
BUG sort ignores command in plain text files

### ISSUE THREAD

**Issue by coldpointblue** [atom/sort-lines#15]
BUG Example:
Step 1 - Make a new plain text file, then paste this...

-----
(seen-AWESOME)237_hd_a_strategy_for_great_work.mov
(seen-EXCELLENT-KEY)session_103__ipad_and_iphone_user_interface_design.mov
(seen-HowToDistribute)modern_tools_and_techniques.mov
(seen-IMPORTANT-BLOCKS)session_712__asynchronous_design_patterns_with_blocks_gcd_and_xpc.mov
(seen-GOOD)227_hd_creating_modern_cocoa_apps.mov

Step 2 - now 'sort'...
... NOTHING HAPPENS, no sort at all.  BUG!

Step 3 - Now make a new plain text file, type a RETURN on the first line, THEN PASTE the same thing...
Step 4 - Now 'sort'. Success, you get this...

-----
(seen-AWESOME)237_hd_a_strategy_for_great_work.mov
(seen-EXCELLENT-KEY)session_103__ipad_and_iphone_user_interface_design.mov
(seen-GOOD)227_hd_creating_modern_cocoa_apps.mov
(seen-HowToDistribute)modern_tools_and_techniques.mov
(seen-IMPORTANT-BLOCKS)session_712__asynchronous_design_patterns_with_blocks_gcd_and_xpc.mov

So, basically something is preventing the sort command from kicking in if the first line starts with a parenthesis (or some other issue, only you know).

CORRECT: The correct behaviour would be to 'sort' the first attempt, without a problem.

**Comment by asbjornu** [comment]
This plugin is not working for me in a `.md` file. Copying and pasting the stuff I want sorted to another tab works; it is sorted as expected. Restarting Atom did not help; the content within the `.md` file will not be sorted. Is there some sort of debug log from this plugin I can have a look at to see why it isn't sorting in this particular file?

**Comment by lee-dohm** [comment]
@coldpointblue I can't reproduce the original bug using Atom v1.6.0-dev-62c1f6a on Mac OS X 10.11.2. If this is still an Issue, let us know.

@asbjornu I also was unable to reproduce the problem you described using Atom v1.6.0-dev-62c1f6a on Mac OS X 10.11.2. Since your problem isn't related to the original, please open a new Issue if this is still a problem.
